nunca-ouvi-falar-de-rust/index.en.org

```diff
@@ -300,81 +300,81 @@ error[E0282]: type annotations needed
 ** This type system brings new ways of expressing ownership
 
    #+reveal: split
-   Which we can't do in other better-known languages
+   Which we can't express in other popular languages
 
-    #+reveal: split
-    [[https://play.rust-lang.org/?gist=1bc78bcf4678616aa01538b6a281f9ed&version=stable][Playground]]
+   #+reveal: split
+   [[https://play.rust-lang.org/?gist=1bc78bcf4678616aa01538b6a281f9ed&version=stable][Playground]]
 
-    #+BEGIN_SRC rust
-      #[derive(Debug)]
-      struct Request {
-          id: i32
-      }
+   #+BEGIN_SRC rust
+     #[derive(Debug)]
+     struct Order {
```

Comment:
Ahhh, também faz sentido isso. Perdão por ter presumido que quis dizer "request"! :)

Reply:
No problem at all! It is a word with double mapping to English (: